core/src/main/java/com/klaytn/caver/kct/kip_wrapper/KIP7Wrapper.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright 2021 The caver-java Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the “License”);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an “AS IS” BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.klaytn.caver.kct.kip_wrapper;
+
+import com.klaytn.caver.Caver;
+import com.klaytn.caver.kct.kip7.KIP7;
+import com.klaytn.caver.kct.kip7.KIP7DeployParams;
+import org.web3j.protocol.exceptions.TransactionException;
+
+import java.io.IOException;
+import java.lang.reflect.InvocationTargetException;
+import java.math.BigInteger;
+import java.util.Map;
+
+/**
+ * Representing a KIP7Wrapper
+ * 1. This class wraps all static methods of KIP7
+ * 2. This class should be accessed via `caver.kct.kip7`
+ */
+public class KIP7Wrapper {
+    /**
+     * A Caver instance
+     * this should be injected from KCTWrapper(= `caver.kct`)
+     */
+    private Caver caver;
+
+    /**
+     * Creates a KIP7Wrapper instance
+     * @param caver A Caver instance
+     */
+    public KIP7Wrapper(Caver caver) {
+        this.caver = caver;
+    }
+
```

Comment:
@aeharvlee Don't we need to add a create()?

Reply:
@sirano11 
I checked the [KCT Layer in Common Architecture](https://kips.klaytn.com/KIPs/kip-34#contract-abi-kct-layer) and yeah there was `create()`.
I think we need to add `create()` for KIP7, KIP17, and KIP37.
Sorry to missed it T_T.

I'll add implementation about it.
Thanks for your feedback.

Reply:
@sirano11 
Hi, I added `create()` methods for `KIP*Wrapper` classes based on your feedback and [KCT Layer Reference](https://kips.klaytn.com/KIPs/kip-34#contract-abi-kct-layer).

Here is the related commit :)
[Add create methods in KIP*Wrapper](https://github.com/klaytn/caver-java/pull/223/commits/0ca485fc7100564e6bea7324de7bc3243d43b498)

Thanks for your feedback again.